src/mailer/index.ts

```diff
@@ -0,0 +1,116 @@
+import * as nodemailer from 'nodemailer'
+import { Transporter } from 'nodemailer'
+import Mail from 'nodemailer/lib/mailer'
+import fs from 'fs'
+import { promisify } from 'util'
+import Mustache from 'mustache'
+import { env, EnvVariable, processEnv } from '../processEnv'
+import { logger } from '../logger'
+import { LoggerTransport } from './loggerTransport'
+import { MailTemplate, templates } from './mailTemplate'
+
+const readFileContent = promisify(fs.readFile)
+
+class Mailer {
+  protected transporter: Transporter | LoggerTransport
+  protected templates = { html: new Map<string, string>([]), text: new Map<string, string>([]) }
```

Comment:
We aren't required to pass `[]` to the `Map` constructor, so let's remove doing so:
```suggestion
  protected templates = { html: new Map<string, string>(), text: new Map<string, string>() }
```

Reply:
Complete